kubernetes/cray-tftp/templates/nf_nat_tftp.yaml

```diff
@@ -41,7 +41,7 @@ spec:
           requests:
             cpu: ".1"
             memory: "20M"
-        image: "{{ $baseChartValues.imagesHost }}/baseos/alpine:3.12"
+        image: "artifactory.algol60.net/docker.io/library/alpine:3"
```

Comment:
I like letting everything but the major version float.

Reply:
Agree, this is doing a very simple action with basic tools, so we shouldn't need to worry about minor releases.